prefix.go

```diff
@@ -459,6 +462,10 @@ func (p *Prefix) availableips() uint64 {
 	if err != nil {
 		return 0
 	}
+	// We don't report more than 2^32 available IPs by design
+	if (ipprefix.IP.BitLen() - ipprefix.Bits) > 32 {
+		return math.MaxInt32
```

Comment:
Do you want to cap to math.Max**U**int32, as this is 2^32 - 1

Reply:
Sadly swagger will only  create a int32 which will not be able to deserialize uint32 max.

Reply:
Ah, ok, I think you said that already.

The only thing I would suggest then is to change "if (ipprefix.IP.BitLen() - ipprefix.Bits) > 32 {" to "if (ipprefix.IP.BitLen() - ipprefix.Bits) > **31** {" and comments to "2^31" to be correct. Or am I missing something?

Reply:
Done